pkg/vpa/vpa.go

```diff
@@ -157,14 +158,19 @@ func (r Reconciler) namespaceIsManaged(namespace *corev1.Namespace) bool {
 	return r.OnByDefault
 }
 
-func (r Reconciler) reconcileControllersAndVPAs(ns *corev1.Namespace, vpas []vpav1.VerticalPodAutoscaler, controllers []Controller) error {
+func (r Reconciler) reconcileControllersAndVPAs(ns *corev1.Namespace, vpas []vpav1.VerticalPodAutoscaler, controllers []Controller, ignoreControllerKind []string) error {
 	defaultUpdateMode, _ := vpaUpdateModeForResource(ns)
 	defaultResourcePolicy, _ := vpaResourcePolicyForResource(ns)
 	defaultMinReplicas, _ := vpaMinReplicasForResource(ns)
 
 	// these keys will eventually contain the leftover vpas that do not have a matching controller associated
 	vpaHasAssociatedController := map[string]bool{}
 	for _, controller := range controllers {
+		// Check if the controller kind is in the ignore list
+		if contains(ignoreControllerKind, controller.Kind) {
+			continue
+		}
+
```

Comment:
We would prefer to use https://github.com/samber/lo for this type of thing rather than writing one-off helper functions.

Reply:
Thanks for your inout on how to improve the PR.
Could you please review the end-to-end test and change to use "github.com/samber/lo" instead the one-off helper function?

Thanks a lot

Reply:
@sudermanjr 
Sorry, I am not so familiar with the work flows (and buttons) in Github - didn't meant to spam you.